craigslist/__init__.py

```diff
@@ -220,8 +226,12 @@ def get_results(self, limit=None, start=0, sort_by=None, geotagged=False):
                 if self.custom_result_fields:
                     self.customize_result(result, row)
 
-                if geotagged and result['has_map']:
-                    self.geotag_result(result)
+                if (geotagged and result['has_map']) or include_details:
+                    soup = self.fetch_content(result['url'])
```

Comment:
Seems to me like this is overriding the definition of `soup` in line 186, isn't it? Please rename to something else (`row_soup`?)